client/src/App.jsx

```diff
@@ -45,4 +45,14 @@ export default class App extends Component {
     );
   }
 
-}
\ No newline at end of file
+}
+
+
+//     // {
+          //   test: /\.css$/,
```

Comment:
You should remove this if you don't need it.

Reply:
OK. Will remove